### PR TITLE
CVE Search v2 - Fixed an issue where cvss score is null

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44839

## Description
Fixed an issue where the cvss score is null.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CVESearch/Integrations/CVESearchV2/CVESearchV2.py [hunk 1]
```diff
@@ -42,9 +42,10 @@ def cve_to_context(cve) -> Dict[str, str]:
     Returns:
         The cve structure.
     """
+    cvss = cve.get('cvss')
     return {
         'ID': cve.get('id', ''),
-        'CVSS': cve.get('cvss', '0'),
+        'CVSS': cvss if cvss else 'N\\A',
```
Comment (guykeller) [inline Packs/CVESearch/Integrations/CVESearchV2/CVESearchV2.py:48]:
```suggestion
        'CVSS': cvss or 'N\\A',
```

**Review by guykeller — APPROVED** [review]
Please see my note

File: Packs/CVESearch/ReleaseNotes/1_0_11.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### CVE Search v2
+- Fixed an issue where the CVSS score is null.
```
Comment (guykeller) [inline Packs/CVESearch/ReleaseNotes/1_0_11.md:4]:
```suggestion
- Fixed an issue where a null CVSS score was handled poorly.
```

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/CVESearch/Integrations/CVESearchV2/CVESearchV2.py (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CVESearch/Integrations/CVESearchV2/CVESearchV2.py]: done

File: Packs/CVESearch/ReleaseNotes/1_0_11.md (hunk 2 above)
Comment (ChanochShayner) [reply Packs/CVESearch/ReleaseNotes/1_0_11.md]: done

**Review by guykeller — APPROVED** [review]

**Comment by guykeller** [comment]
@ChanochShayner You can ignore the RN test. Approved by me.

**Pull request merged**